src/components/filters/RangeFilter/index.jsx

```diff
@@ -145,6 +145,9 @@ class RangeFilter extends React.Component {
   }
 
   render() {
+    if (this.props.max === 0 && this.props.hideZero) {
```

Comment:
I don't think this is logically correct, what if the input value is `[-1, 0]`?

To implement this `hideZero` for range filter would require more thinking: for single select filter if `hideZero` is true then the filter will be hidden if `count == 0`. But for range filter, we need more logic to replicate this behavior, since the range filter only knows `min` `max` `value` and `rangeStep`

IMO we should also change some logic in Guppy's filter component to implement this: when querying Guppy, it will return `histogran: [null, null], count: 0` for a numerical field, and that is the place we can decide if a range filter should be hidden

Reply:
The correct logic here should be something like 
this.props.max === this.props.min

Do you think i should scrap this approach and instead look into changing what the API returns or handle this after the api returns? do we even have access to make api changes?